src/main/java/com/joyent/manta/monitor/servlets/MantaMonitorServlet.java

```diff
@@ -1,58 +0,0 @@
-/*
```

Comment:
Why did this file get deleted?

Reply:
It was there to just expose the elapsed time on a /monitor api, which I was planning to remove anyways, since it is not used anywhere. We will have only the /metrics exposed to the user.